Packs/SecurityScorecard/Integrations/SecurityScorecard/SecurityScorecard.py

```diff
@@ -0,0 +1,1078 @@
+            url_suffix='companies/{0}/factors'.format(domain),
+            params=request_params
+        )
+
+    def get_company_historical_scores(self, domain: str, _from: str, to: str, timing: str) -> Dict[str, Any]:
+
+        request_params: Dict[str, Any] = {}
+
+        if _from:
+            request_params['from'] = _from
+
+        if to:
+            request_params['to'] = to
+
+        if timing:
+            request_params['timing'] = timing
+        # API by default is set to daily
+        else:
+            request_params['timing'] = 'daily'
+
+        return self._http_request(
+            'GET',
+            url_suffix='companies/{0}/history/score'.format(domain),
+            params=request_params)
+
+    def get_company_historical_factor_scores(self, domain: str, _from: str, to: str, timing: str) -> Dict[str, Any]:
+
+        request_params: Dict[str, Any] = {}
+
+        if _from:
+            request_params['from'] = _from
+        if to:
+            request_params['to'] = to
+        if timing:
+            request_params['timing'] = timing
+
+        return self._http_request(
+            'GET',
+            url_suffix='companies/{0}/history/factors/score'.format(domain),
+            params=request_params
+        )
+
+    def create_grade_change_alert(
+        self,
+        email: str,
+        change_direction: str,
+        score_types: List[str],
+        target: List[str]
+    ) -> Dict[str, Any]:
+
+        payload: Dict[str, Any] = {}
+        if change_direction:
+            payload["change_direction"] = change_direction
+
+        if len(score_types) > 0:
+            payload["score_types"] = score_types
+
+        if len(target) > 0:
+            payload["target"] = target
+
+        return self._http_request(
+            'POST',
+            url_suffix="users/by-username/{0}/alerts/grade".format(email),
+            json_data=payload
+        )
+
+    def create_score_threshold_alert(
+        self,
+        email: str,
+        change_direction: str,
+        threshold: int,
+        score_types: List[str],
+        target: List[str]
+    ) -> Dict[str, Any]:
+
+        payload: Dict[str, Any] = {}
+        if change_direction:
+            payload["change_direction"] = change_direction
+
+        threshold_arg = arg_to_number(arg=threshold, arg_name='threshold', required=True)
+        payload["threshold"] = threshold_arg
+
+        if len(score_types) > 0:
+            payload["score_types"] = score_types
+
+        if len(target) > 0:
+            payload["target"] = target
+
+        return self._http_request(
+            'POST',
+            url_suffix="users/by-username/{0}/alerts/score".format(email),
+            json_data=payload
+        )
+
+    def delete_alert(self, email: str, alert_id: str, alert_type: str) -> None:
+
+        return self._http_request(
+            "DELETE",
+            url_suffix="users/by-username/{0}/alerts/{1}/{2}".format(email, alert_type, alert_id),
```

Comment:
In the docs I see the path is with the string `grade`: `https://api.securityscorecard.io/users/by-username/username/alerts/grade/alert`

Reply:
See [comment above](https://github.com/demisto/content/pull/13582#discussion_r672983505). Resolving this conversation as we can discuss implementation method in aforementioned comment.